actions/building_positioning.py

```diff
@@ -13,7 +13,8 @@ class building_positioning:
         ressources = self.state.mineral_field.closer_than(10, start)
         behind_ressources = [point for point in all_points if 2 < point.distance_to(ressources.closest_to(point)) < 4]
         for point in behind_ressources:
-            if await self.can_place(EVOLUTIONCHAMBER, point):
+            # also check engineering bay placement for hatcheries that just spawned but have no creep around
+            if await self.can_place(ENGINEERINGBAY, point) or await self.can_place(EVOLUTIONCHAMBER, point):
```

Comment:
I think you put engineering bay to test, but I think it can be removed now

Reply:
No, i tried to explain you already, but its very tricky. When we get into this function with an expansion hatchery, we cant check for building places around it because the can place function also checks for creep. Thats why we have to check if we can place an ebay around it, with the same size but not on creep, to see if we can place the evo later